doc/recipe/preprocessor.rst

```diff
@@ -1475,6 +1476,28 @@ Parameters:
     ``'nearest'``. There is no default.
 
 
+``extract_location``
+--------------------
+
+Extract a single point using a location name, with interpolation
+(either linear or nearest). This preprocessor extracts a single
+location point from a cube, according to the given interpolation
+scheme ``scheme``. The function retrieves the coordinates of the
+location and then calls the :func:`esmvalcore.preprocessor.extract_point`
+preprocessor. It can be used to locate cities and villages,
+but also mountains or other geographical locations.
+
+Note that this function's geolocator application needs a
+working internet connection.
```

Comment:
```suggestion
.. note::
   Note that this function's geolocator application needs a
   working internet connection.
```

Reply:
yeh OK lots other functions have in-line notes